File: Assignment 3/primenumber/src/primenumber/Primenumber.java

package primenumber;

public class Primenumber {

    
    public static void main(String[] args) {
        int n=17,temp;
        boolean isPrime=true;
        for(int i=2;i<=n/2;i++)
        {
            temp=n%i;
            if(temp==0)
            {
                isPrime=false;
                break;
                
            }
            
        }
        if(isPrime)
        {
            System.out.println(n + "is a prime number");
        }
        else
            System.out.println(n + "is not a prime number");
    }
    
}

File: Assignment 3 in c#/PrimeNumber.cs
using System;  
  public class PrimeNumber
   {  
     public static void Main(string[] args)  
      {  
          int n, i, m=0, l=0;    
          Console.Write("Enter the Number to check Prime: ");    
          n = int.Parse(Console.ReadLine());  
          m=n/2;    
          for(i = 2; i <= m; i++)    
          {    
           if(n % i == 0)    
            {    
             Console.Write("Not a Prime Number");    
             l=1;    
             break;    
            }    
          }    
          if (l==0)    
           Console.Write("Number is Prime.");       
     }  
   }  